pmd-swift/src/test/java/net/sourceforge/pmd/lang/swift/RuleSetFactoryTest.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.swift;
+
+import net.sourceforge.pmd.AbstractRuleSetFactoryTest;
+
+class RuleSetFactoryTest extends AbstractRuleSetFactoryTest {
```

Comment:
I wonder if these test classes should be renamed. I think a name like "SwiftStandardRulesetsTests" would be more appropriate, RuleSetFactory is in pmd-core and is deprecated.

Reply:
Yes, probably. I think, we have this for most languages and it's also part of the howto (https://docs.pmd-code.org/pmd-doc-7.0.0-SNAPSHOT/pmd_devdocs_major_adding_new_language_javacc.html#15-test-the-rules). It really would help to use a naming pattern like your suggestion.